src/js/client/coverageSender.js

```diff
@@ -80,4 +85,4 @@
             QUnit.done(afterTestsCallback);
         }
     };
-})(typeof Testem !== "undefined" ? Testem : false, QUnit);
+})(typeof Testem !== "undefined" ? Testem : false, typeof QUnit !== "undefined" ? QUnit : false);
```

Comment:
Perhaps just window.Testem and window.QUnit is a touch more succinct?

Reply:
Agreed, and fixed.